numpy/core/tests/test_simd.py

```diff
@@ -331,12 +331,13 @@ def test_square(self):
         assert square == data_square
 
     @pytest.mark.parametrize("intrin, func", [("self.ceil", math.ceil),
-    ("self.trunc", math.trunc)])
+    ("self.trunc", math.trunc), ("self.floor", math.floor)])
     def test_rounding(self, intrin, func):
         """
         Test intrinsics:
             npyv_ceil_##SFX
             npyv_trunc_##SFX
+            npyv_floor##SFX
```

Comment:
we need a test case for `-0.0 == -0.0`, you have disabled the whole signed-zero tests within #20367